pkg/controller/core/queuedworkload_controller.go

```
@@ -142,27 +152,39 @@ func (r *QueuedWorkloadReconciler) Update(e event.UpdateEvent) bool {
 	case prevStatus == pending && status == pending:
 		if !r.queues.UpdateWorkload(oldWl, wl.DeepCopy()) {
```

ditto, we can return an error indicating what was missing